src/ft/mod.rs

```diff
@@ -17,10 +17,12 @@ pub mod fc;
 use fc::{CharSet, FTFaceLocation, Pattern, PatternHash, PatternRef, Rgba};
 
 use super::{
-    BitmapBuffer, FontDesc, FontKey, GlyphKey, Metrics, Rasterize, RasterizedGlyph, Size, Slant,
-    Style, Weight,
+    BitmapBuffer, Error, FontDesc, FontKey, GlyphKey, Metrics, Rasterize, RasterizedGlyph, Size,
+    Slant, Style, Weight,
 };
 
+const MISSING_GLYPH_INDEX: u32 = 0;
```

Comment:
Is this documented somewhere?

Reply:
Well, yes, right on a method we're using to get a char index.